src/js/views/forms/form/widgets/widget_header.scss

```diff
@@ -0,0 +1,12 @@
+.form__widgets {
```

Comment:
I think we should probably change this file name to `form-widgets.scss` and this class to `form-widgets` and `form-widgets__section`. That way when you see the class name you can find the file by the block

Reply:
Will get that updated!